app/assets/stylesheets/content/_forms.sass

```diff
@@ -187,6 +161,38 @@ fieldset.form--fieldset
 
 .form--fieldset-legend
   @extend %form--fieldset-legend-or-section-title
+  width:     100%
+
+  .form--fieldset.-collapsible > &
+    @include without-link-styling
+    cursor:  pointer
+
+  &:after
+    @include icon-common
+    float:   right
+    padding:
+      right: 5px
+      top:   4px
+
+    .form--fieldset.-collapsible > &
+      content: "\e0d4"
+
+    .form--fieldset.-collapsible.-collapsed > &,
+    .form--fieldset.-collapsible.collapsed  > &
+      content: "\e0d2"
+
+.form--fieldset-control
+  float:          right
+  margin-top:     -2.8rem
+  text-align:     right
+  color:          lighten($body-font-color, 10)
+  font-size:      1rem
+  font-style:     italic
+  line-height:    1.8
+
+.form--fieldset-control-container
+  padding: 0 0.25rem
+  background-color: white
```

Comment:
Is this intended? It looks awkward:

![white-bg](https://cloud.githubusercontent.com/assets/453584/7115827/4c12a632-e1ec-11e4-8ee4-51502c1202e5.PNG)

Btw. the living Styleguide is great!
I can search for "fieldset" and look for all occurences of fieldsets that might have been broken by @myabc :)